docker/sandbox/kustomization.yaml.tmpl

```diff
@@ -0,0 +1,9 @@
+
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+  - flyte_generated.yaml
+
+# Image overrides
```

Comment:
Probably unnecessary since this file is not going to be modified directly right? Is the idea to mount a modified file here to override the default sandbox config?

Reply:
yes, I will mount the user-provided kustomize file and the entry point script will handle the logic. By default, it will not override anything.